.env

```diff
@@ -1 +1,2 @@
 REACT_APP_API_VERSION=v3.5.0
+GOOGLE_RECAPTCHA_SITE_KEY=6LdENZInAAAAAP9c5t9mr-yQ9vL4NObSBBgrvrC7
```

Comment:
Just double-checking here: the site key is supposed to be public, right? I just want to make sure this is not the private key (secret).

Reply:
I'd rather the .env wasn't being pushed up but this particular key is not sensitive. The secret key is on the API side and will be stored encrypted